# This represents a locomotion trajectory with reference coordinate frame and timestamp
Header header

# The current locomotion state
dwl_msgs/WholeBodyState actual

# The list of reported locomotion trajectories
dwl_msgs/WholeBodyState[] trajectory
